swingset-extensions/load-sample-mdx/index.ts

```diff
@@ -0,0 +1,36 @@
+import * as fs from 'fs'
+import * as path from 'path'
+
+import { serialize } from 'next-mdx-remote/serialize'
+import { anchorLinks } from '@hashicorp/platform-remark-plugins'
+import jumpToSection from '@hashicorp/platform-remark-plugins/plugins/jump-to-section'
+
+import { GetStaticPropsContext } from 'next'
+
+const loadSampleMdx = async (ctx: GetStaticPropsContext, staticProps) => {
+  if (ctx.params.swingset?.includes('docspage')) {
+    const sampleMdxSource = fs.readFileSync(
+      path.join(process.cwd(), 'packages/docs-page/__fixtures__/sample.mdx'),
+      'utf-8'
+    )
+
+    const headings = []
+    const { compiledSource, scope } = await serialize(sampleMdxSource, {
+      mdxOptions: {
+        remarkPlugins: [[anchorLinks, { headings }], jumpToSection],
+      },
+      scope: { headings },
+    })
+
+    staticProps.props.mdxSource.scope.componentProps.staticProps.properties.mdxSource.testValue = {
+      compiledSource,
+      scope,
+    }
```

Comment:
😱  This is kinda crazy haha

Reply:
😐 Yea, that's just the current state of the output of [`createPage`](https://github.com/hashicorp/swingset/blob/main/page.jsx#L14)

Not sure what the ideal approach here is... but rendering content from a real `.mdx` file is a massive improvement over hard-coding the string content, prior to this PR

Reply:
Definitely, no disagreement there. I think there's room for continued iteration here; I see this as the first step towards supporting server-portions of some of our components. Nice job!